src/BasicDarcInfo/Program.cs

```diff
@@ -31,10 +31,6 @@
 app.UseHttpsRedirection();
 app.UseStaticFiles();
 
-app.UseRouting();
```

Comment:
Why can this be removed? Is it because `MapGet` is implicitly adding routing support later?

Reply:
The `WebApplication` construct automatically registers certain middleware, routing being one of them, so you don't need to include them in app code. _Unless_ you want to enforce some particular ordering for middlewares, like run Middleware A before route matching happens, then you can register both in your app in the desired order and `WebApplication` won't register the middleware again.